src/snowflake/snowpark/column.py

```diff
@@ -826,6 +826,7 @@ def bitxor(
 
     # Note: For the operator overrides we always emit ast, it simply gets ignored in a call chain.
 
+    @publicapi
```

Comment:
Dunder/magic methods won't accept additional arguments afaik, so I don't think we should decorate them. Is this working in our tests?

Reply:
+1, no need to decorate dunder methods.